### DISCUSSION TITLE
I'm an idiot, how do I install this thing as a VST for Ableton?

### DISCUSSION THREAD

**Discussion by VictorLincolnPine** [DISTRHO/Cardinal#507] [Q&A]
Like it says on the tin, I have no idea how to install Cardinal as a VST plugin for Ableton. Can I get pointed to where an installation instruction manual is or something? I find presentation videos, but all assuming that Cardinal is installed already and looking through frequently asked questions leads me to a page full of text that leaves my eyes blurry and my brain hurting.

**Comment by falkTX** [comment]
you get the installer (windows or macos), install cardinal and restart ableton.
some details on https://github.com/DISTRHO/Cardinal/wiki/Install but it is really just picking the installer from the latest release, for now that being https://github.com/DISTRHO/Cardinal/releases/tag/23.02

**Reply by VictorLincolnPine** [reply]
thank you
